src/IntegrationTests/java/uk/gov/hmcts/reform/sscs/callback/AdjournCaseIt.java

```diff
@@ -36,482 +37,372 @@
+        "callback/adjournCaseGeneratedPaperWhenCaseNotListedStraightAwayWithoutDirectionsMade.json";
+    public static final String GENERATED_FACE_TO_FACE_WHEN_CASE_NOT_LISTED_STRAIGHT_AWAY_WITHOUT_DIRECTIONS_MADE_JSON =
+        "callback/adjournCaseGeneratedFaceToFaceWhenCaseNotListedStraightAwayWithoutDirectionsMade.json";
+    public static final String VALID_SUBMISSION_WITH_SET_GENERATED_DATE_JSON =
+        "callback/adjournCaseValidSubmissionWithSetGeneratedDate.json";
+    public static final String MANUALLY_GENERATED_JSON = "callback/adjournCaseManuallyGenerated.json";
+    public static final String GENERATED_FACE_TO_FACE_WITH_INTERPRETER_REQUIRED_AND_LANGUAGE_SET_JSON =
+        "callback/adjournCaseGeneratedFaceToFaceWithInterpreterRequiredAndLanguageSet.json";
+    public static final String GENERATED_FACE_TO_FACE_WITH_INTERPRETER_REQUIRED_AND_LANGUAGE_NOT_SET_JSON =
+        "callback/adjournCaseGeneratedFaceToFaceWithInterpreterRequiredAndLanguageNotSet.json";
+    public static final String GENERATED_FACE_TO_FACE_WITH_INTERPRETER_NOT_REQUIRED_AND_LANGUAGE_SET_JSON =
+        "callback/adjournCaseGeneratedFaceToFaceWithInterpreterNotRequiredAndLanguageSet.json";
+    public static final String GENERATED_TELEPHONE_WHEN_CASE_NOT_LISTED_STRAIGHT_AWAY_WITHOUT_DIRECTIONS_MADE_JSON =
+        "callback/adjournCaseGeneratedTelephoneWhenCaseNotListedStraightAwayWithoutDirectionsMade.json";
+    public static final String GENERATED_VIDEO_WHEN_CASE_NOT_LISTED_STRAIGHT_AWAY_WITHOUT_DIRECTIONS_MADE_JSON =
+        "callback/adjournCaseGeneratedVideoWhenCaseNotListedStraightAwayWithoutDirectionsMade.json";
+    public static final String CCD_ABOUT_TO_SUBMIT = "/ccdAboutToSubmit";
+    public static final String CCD_MID_EVENT_PREVIEW_ADJOURN_CASE = "/ccdMidEventPreviewAdjournCase";
+    public static final String CCD_MID_EVENT_ADJOURN_CASE_POPULATE_VENUE_DROPDOWN = "/ccdMidEventAdjournCasePopulateVenueDropdown";
+    public static final String CCD_MID_EVENT = "/ccdMidEvent";
+    public static final String TEST_NAME = "An Test";
+    public static final String CASE_ID = "12345656789";
+    public static final String NINO = "JT 12 34 56 D";
+    public static final String DRAFT_ADJOURNMENT_NOTICE = "DRAFT ADJOURNMENT NOTICE";
+    public static final String CHESTER_MAGISTRATE_S_COURT = "Chester Magistrate's Court";
+    public static final String FIRST_MORNING_SESSION_ON_A_DATE_TO_BE_FIXED =
+        "It will be first in the morning session on a date to be fixed";
+    public static final String FACE_TO_FACE_HEARING = "face to face hearing";
+    public static final String A_STANDARD_TIME_SLOT = "a standard time slot";
+    public static final String JUDGE_FULL_NAME = "Judge Full Name";
+    public static final String FACE_TO_FACE = "faceToFace";
+    public static final String SOMETHING_ELSE = "something else";
+    public static final String REASONS_1 = "Reasons 1";
```

Comment:
(minor) I think some of these are a bit much in terms of constants, for instance, REASONS_1 is only used once along with SOMETHING_ELSE whereas DOCUMENT_URL is okay as its used in both a when(...) and a assertion

Reply:
I got over zealous with extracting all the strings to constants! Will revert those which only appear once.